src/annotator/config/settings.js

```diff
@@ -1,6 +1,35 @@
 'use strict';
 
 /**
+ * Return the href URL of the first annotator link in the given document.
+ *
+ * Return the value of the href attribute of the first
+ * `<link type="application/annotator+html">` element in the given document.
+ *
+ * This URL is used as the src of the sidebar's iframe.
+ *
+ * @param {Document} - The document to search.
+ * @return {string} - The URL to use for the sidebar's iframe.
+ *
+ * @throws {Error} - If there's no annotator link or the first annotator has
+ *   no href.
```

Comment:
Previously the code would throw this error, but the fact wasn't documented or tested. (Not hugely important - in practice the link is always there.)